src/components/models/Node.ts

```diff
@@ -10,6 +10,10 @@ interface Node {
   count?: number;
   total?: number;
   gatewayUID: string;
+  /**
+   * The signal strength for this node - lora bars.
+   */
+  bars?: number;
```

Comment:
A bit strange that `bars` is a number but `cellBars` and `wifiBars` are strings. An enum might work well here since there are only a few different states for each, and I think that would still serialize ok.